LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java

```diff
@@ -127,7 +97,8 @@ public void runTestIterations(String[] args, String repo, String commit, int com
             System.out.println(attempt);
 
             if (isStart){
-                resultAnalyzer = startTesting(app, "1", args, repo, attempt, CSVCreator, resultAnalyzer);
+                resultAnalyzer = startTesting("1", args, repo, attempt, CSVCreator, resultAnalyzer);
+                isStart = false;
                 if (!this.badResponse){
                     isStart = false;
                 }
```

Comment:
Were you wanting to have isStart for both cases?

Reply:
Oops that was coz of a merge conflict. I'll fix it now.